app/Components/Statistics/TotalStatistics.tsx

```diff
@@ -3,14 +3,21 @@ import { Text, useTheme } from "react-native-paper";
 import { useWindowDimensions } from "react-native";
 import Statistic from "./Statistic";
 import { formatTime } from "../SudokuBoard/Functions/BoardFunctions";
+import { SudokuStrategy } from "sudokuru";
+import React from "react";
+import { formatOneLessonName } from "../../Functions/learnedLessons";
 
-interface TotalStatisticsProps {
+export interface TotalStatisticsProps {
```

Comment:
would be nice to have a comment explaining what this file does, e.g. might confuse end of game stats and stats page code